app/javascript/dashboard/routes/dashboard/settings/agents/EditAgent.vue

```diff
@@ -65,6 +83,8 @@ import WootSubmitButton from '../../../../components/buttons/FormSubmitButton';
 import Modal from '../../../../components/Modal';
 import Auth from '../../../../api/auth';
 
+const AVAILABILITY_STATUS_KEYS = ['online', 'busy', 'offline'];
```

Comment:
can we move this to `dashboard/constants.js`?

Reply:
Sure.